static/js/webapp.js

```diff
@@ -201,6 +201,19 @@ function select_all(bool) {
     }
 }
 
+function clean_up_imgs() {
+    if (confirm("Are you sure you wish to delete your uploaded images?")) {
```

Comment:
UX rule 1 (http://babich.biz/ux-writing/) if you can say it in fewer words...

"Delete all uploaded images?"

Reply:
I can change it, I was going for the "Are you sure..." from OS empty trash prompts as a kind of joke, but now that this text is on the main page and not the admin page it should probably be clearer.